cabal.project

```diff
@@ -86,6 +86,12 @@ package yet-another-logger
 --
 --   nix-prefetch-git --url <location> --rev <tag>
 
+source-repository-package
+    type: git
+    location: https://gitlab.com/edmundnoble/property-matchers.git
```

Comment:
Do we have other gitlab reps?

Reply:
This is just mine :innocent: I'll delete this entry because it should be on hackage by now.